lib/workspace.js

```diff
@@ -369,9 +379,38 @@ module.exports = {
       localPathExists = false;
     }
 
+    const dynamicFiles = Array.isArray(variant_params?._files)
+      ? variant_params._files
+          .map((file) => {
+            try {
+              if (!file.name) {
+                logger.warn(`Dynamic file included without a name. Ignoring file.`);
```

Comment:
I'd be inclined to remove these logs, as they won't ever be seen by users and will just serve to clutter up our normal logs. We should think of some way to surface these to instructors - perhaps via a course issue? We don't currently have any great way to show an error that was produced when a workspace was being initialized.

Reply:
I took a look at the option of creating an issue, but it would require a few changes that I'm not comfortable doing now. One example is the fact that the initialization is done in the context of a socket connection, where the authn_user_id is not readily available, and that field is required in the issue. I changed it to a comment for now. I can add a GitHub issue to track this.

Reply:
A comment is fine for now, yeah.